commercial/app/views/books/book.scala.html

```diff
@@ -1,5 +1,5 @@
-@(book: model.commercial.Book, omnitureId: Option[String], clickMacro: Option[String])(implicit request: RequestHeader)
-@import views.support.{ImgSrc, Item300}
+@import _root_.commercial.model.merchandise.Book
```

Comment:
But if commercial has now been removed from `model`, then this won't happen? Did you verify by removing the import?

It might be because of the project `commercial` gets a namespace of it's own during compilation. I have come up against this before (although not the exact same situation), it's very frustrating to know where the other reference is coming from. You could print it out!